docs/program/extend/modular-resources/add-rplidar-module.md

```diff
@@ -78,6 +78,12 @@ Now, add the Rplidar as a modular component of your robot in the [Viam app](http
 
   ```json
   {
+    "modules": [
+      {
+        "executable_path": "/Users/<YOUR_USERNAME>/Library/Caches/Homebrew/rplidar-module--git",
```

Comment:
This seems odd to me. For example, I don't have that file on my computer (macOS x86_64) even though I have the rplidar-module installed. I also just confirmed that running `brew install rplidar-module` installs the module at `/usr/local/bin/rplidar-module`. 

Why are you suggesting this change?

Reply:
@kkufieta No need, this is inaccurate-- Will keep at usr/local/bin/rplidar-module

Reply:
Thanks for jumping on this so quickly-- I could only find reference to the module on my local machine in the brew Caches when testing brew install on my own, so I wanted to get review and check where it should be. 
I still don't have the module at /usr/local/bin, which is odd

Reply:
That's weird, indeed. If you'd like to, we can look at this together tomorrow!